chromium_src/tools/web_dev_style/eslint.py

```diff
@@ -21,6 +21,7 @@ def Run(_original_function, **kwargs):
         node_args += ['--fix']
     # Don't ignore files/directories starting with a dot.
     node_args += ['--ignore-pattern', '!.*']
-    node_args += kwargs['args']
+    # Ignore the .eslintrc.js file - it breaks when we include it.
+    node_args += [x for x in kwargs['args'] if not '.eslintrc.js' in x]
```

Comment:
@goodov I originally tried adding another rule to `ignore-pattern` but it didn't help :/